pandas/tests/generic/test_series.py

```diff
@@ -261,3 +256,15 @@ def test_to_xarray(self):
         tm.assert_almost_equal(list(result.coords.keys()), ["one", "two"])
         assert isinstance(result, DataArray)
         tm.assert_series_equal(result.to_series(), s)
+
+    @td.skip_if_no("xarray", min_version="0.7.0")
```

Comment:
cc @jbrockmendel your test moving PR for xrray might need rebasing after this

Reply:
actually i think we *always* have a > 0.7.0 xarary?

Reply:
I can check that as a follow-up. There are more spots where this condition is used